cypress/page/shoping-cart.page.ts

```diff
@@ -3,7 +3,7 @@ class ShopingCart {
 
     constructor(){
         // There's no need to improve this selector.
-        this.proceedToCheckoutButton =  ".cart_navigation span"
+        this.proceedToCheckoutButton =  ".cart_navigation a[title='Proceed to checkout']"
```

Comment:
just `a[title='Proceed to checkout']` should work

Reply:
that selector matchs with two results, so i couldn't change it :/

Reply:
![image](https://user-images.githubusercontent.com/46982083/181926386-ea7ded15-4910-4876-8bc8-542063085ab7.png)

Reply:
ok